### PR TITLE
feat: custom relayer card rpc which uses rpc call

### PR DESCRIPTION
**Pull request by stizzle123**
Links: https://app.clickup.com/t/2tdm0ry
Screenshot 2022-08-15 at 18 55 58

### CODE REVIEW

**Review by andyogaga — APPROVED** [review]

**Review by dzbo — CHANGES_REQUESTED** [review]
There should ba a way to add `chainIds` as well. It could be a token input field with preselected first value with current chain but should also allow to add other ones. Something like we use for UP tags would work I guess:

CleanShot 2022-08-16 at 09 45 02@2x

**Comment by stizzle123** [comment]
> There should ba a way to add `chainIds` as well. It could be a token input field with preselected first value with current chain but should also allow to add other ones. Something like we use for UP tags would work I guess:
> 
> CleanShot 2022-08-16 at 09 45 02@2x

Yeah, I'll update now

File: src/components/endpoints/CustomRelayer.vue
Comment (Hugoo) [inline src/components/endpoints/CustomRelayer.vue:86]: Do we need to keep this?

Comment (dzbo) [reply src/components/endpoints/CustomRelayer.vue:86]: ✂️

Comment (stizzle123) [reply src/components/endpoints/CustomRelayer.vue]: Yeah, just added a style

**Review by Hugoo — APPROVED** [review]

**Comment by dzbo** [comment]
The fields should be disabled before you connect dApp to extension. 

btw, please don't merge PR's before they are approved 😅

**Pull request merged**
